src/pycram/datastructures/dataclasses.py

```diff
@@ -25,6 +25,31 @@
     from .world_entity import PhysicalBody
 
 
+@dataclass
+class ManipulatorData:
```

Comment:
Could you add some short doc to the parameters, that would make it easier to define them